pkg/daemon/ceph/osd/volume.go

```diff
@@ -151,6 +153,15 @@ func updateLVMConfig(context *clusterd.Context) error {
 }
 
 func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceOsdMapping) error {
+	versionString, versionErr := context.Executor.ExecuteCommandWithOutput(false, "", "ceph", "--version")
```

Comment:
Please don't do this, use `a` has cluster which has `ClusterInfo` and `CephVersion`. It should already have the correct Ceph version so let's use it.